webapp/app/[locale]/stake/_components/stakeAndEarnPointsSpecialLabel.tsx

```diff
@@ -0,0 +1,82 @@
+import { useTranslations } from 'next-intl'
+
+import { StarIcon } from './icons/star'
+
+interface Props {
+  gradientMode?: 'primary' | 'secondary'
+  textSize?: 'text-6xl' | 'text-4xl'
+}
+
+export function StakeAndEarnPointsSpecialLabel({
+  gradientMode = 'primary',
+  textSize = 'text-6xl',
+}: Props) {
+  const t = useTranslations('stake-page.dashboard')
+  const labelParts = t('stake-earn-points-label').split('|')
+
+  // This function is designed to render a gradient label with a special behavior for the letter 'i'.
+  // If the second part of the label (labelParts[1]) contains the letter 'i', a star icon is rendered
+  // above it. However, in cases where the translation does not include the letter 'i' (e.g., in Spanish),
+  // a conditional logic ensures that the label is rendered correctly without the star icon.
+
+  const gradients = {
+    primary: {
+      firstText: 'from-[#FF6F3D] via-[#FF8B4F] to-[#FFD37C]',
+      secondText: 'from-[#FF7849] via-[#FF9A5A] to-[#FFC86B]',
+      starColor: '#FF7849',
+    },
+    secondary: {
+      firstText: 'from-[#DD3818] via-[#e46830] to-[#ea8d61]',
+      secondText: 'from-[#dd3818] via-[#e46830] to-[#ea8d61]',
+      starColor: '#e46830',
+    },
+  }
+
+  const selectedGradients = gradients[gradientMode]
+
+  function getStarIconClassName() {
+    if (textSize === 'text-6xl') {
+      return 'absolute -top-[54px] left-[9px] -translate-x-1/2'
+    }
+    if (textSize === 'text-4xl') {
+      return 'absolute -top-[35px] left-[5px] -translate-x-1/2'
```

Comment:
Same as [this comment](https://github.com/hemilabs/ui-monorepo/pull/800/files#r1935676607), does this extra pixel in each direction make a difference visually?

```suggestion
      return 'absolute -top-9 left-2 -translate-x-1/2'
```

This will create extra CSS classes, it's ok if needed, but I am wondering if it is truly the case 😄

Reply:
I've done some refactor to use group variant so that way we can get closer to the tailwind philosophy.